### PR TITLE
Fix end content padding of Chip

### PR DESCRIPTION
**Pull request by luizgrp**
#### WHAT

Fix end content padding of `Chip`.

#### WHY

In order to align with specs.

#### Checklist :clipboard:
- [N/A] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [N/A] Update metalava's signature text files

### CODE REVIEW

File: compose-material/src/main/java/com/google/android/horologist/compose/material/Chip.kt [hunk 1]
```diff
@@ -200,9 +200,14 @@ public fun Chip(
         }
 
     val contentPadding = if (largeIcon) {
-        val horizontalPadding = 10.dp
-        val verticalPadding = 6.dp // same as Chip.ChipVerticalPadding
-        PaddingValues(horizontal = horizontalPadding, vertical = verticalPadding)
+        val horizontalPadding = 14.dp // same as ChipDefaults.ChipHorizontalPadding
```
Comment (yschimke) [inline compose-material/src/main/java/com/google/android/horologist/compose/material/Chip.kt:203]: Should these be public?

Comment (yschimke) [reply compose-material/src/main/java/com/google/android/horologist/compose/material/Chip.kt:203]: Also, may as well inline if it's only the end padding?

**Review by yschimke — APPROVED** [review]

File: compose-material/src/main/java/com/google/android/horologist/compose/material/Chip.kt (hunk 1 above)
Comment (luizgrp) [reply compose-material/src/main/java/com/google/android/horologist/compose/material/Chip.kt:203]: I can raise a feature request to ask them to be public.

What would be the inline version?

Comment (luizgrp) [reply compose-material/src/main/java/com/google/android/horologist/compose/material/Chip.kt:203]: b/287599459

File: compose-material/src/main/java/com/google/android/horologist/compose/material/Chip.kt [hunk 2]
```diff
@@ -200,9 +200,14 @@ public fun Chip(
...
+        val verticalPadding = 6.dp // same as ChipDefaults.ChipVerticalPadding
+        PaddingValues(
+            start = 10.dp,
+            top = verticalPadding,
+            end = horizontalPadding,
+            bottom = verticalPadding
```
Comment (yschimke) [inline compose-material/src/main/java/com/google/android/horologist/compose/material/Chip.kt:209]:
```suggestion
        val verticalPadding = 6.dp // same as ChipDefaults.ChipVerticalPadding
        PaddingValues(
            start = 10.dp,
            top = verticalPadding,
            end = 14.dp, // same as ChipDefaults.ChipHorizontalPadding
            bottom = verticalPadding
```

**Pull request merged**